modules/monitoring-searching-alerts-silences-and-alerting-rules.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+[id="searching-alerts-silences-and-alerting-rules_{context}"]
+= Searching and filtering alerts, silences, and alerting rules
+
+You can filter the alerts, silences and alerting rules that are displayed in the Alerting UI. This section provides a description of each of the available filtering options.
+
+[discrete]
+== Understanding alert filters
+
+In the Administrator perspective, the *Alerts* page in the Alerting UI provides details about alerts relating to default {product-title} and user-defined projects. The page includes a summary of each alert's severity, state, and source. The time at which an alert went into its current state is also shown.
+
+You can filter by alert state, severity and source. By default, only *Platform* alerts that are *Firing* are displayed. The following describes each alert filtering option:
+
+* *Alert State* filters:
+** *Pending*. The alert is active but is waiting for the duration that is specified in the alerting rule before it fires.
+** *Firing*. The alert is firing because the alert condition is true and the optional `for` duration has passed. The alert will continue to fire as long as the condition remains true.
+** *Silenced*. The alert is now silenced for a defined time period. Silences temporarily mute alerts based on a set of label selectors that you define. Notifications will not be sent for alerts that match all the listed values or regular expressions.
+
+* *Severity* filters:
+** *Critical*. The condition that triggered the alert could have a critical impact. The alert requires immediate attention when fired and is typically paged to an individual or to a critical response team.
+** *Warning*. The alert provides a warning notification about something that might require attention in order to prevent a problem from occurring. Warnings are typically routed to a ticketing system for non-immediate review.
+** *Info*. The alert is provided for informational purposes only.
+** *None*. The alert has no defined severity.
+** You can also create custom severity definitions for alerts relating to user-defined projects.
+
+* *Source* filters:
+** *Platform*. Platform-level alerts relate only to default {product-title} projects. These projects provide core {platform-title} functionality.
```

Comment:
It doesn't look like `platform-title` is defined, as it is showing up as `{platform-title}` verbatim in the preview.

Reply:
Good catch! I have subsequently changed this to `{product-title}`.